js/effects.js

```diff
@@ -0,0 +1,10 @@
+//effects objects
+let effects = {
```

Comment:
I think I'm a little confused on the difference between the effects.js and the timers js. can I get some clarification on what each script's main purpose is and how they differ from each other? Thanks!

Reply:
effects.js is a list of all effects and their properties, like a SST. timers.js is the list of current timers for the active effects, and it draws from the SST for that particular effect.